docs/jaeger-support.asciidoc

```diff
@@ -53,6 +53,27 @@ Based on the endpoint enabled in the previous step, configure the relevant host
 * `apm-server.jaeger.grpc.host` defaults to `localhost:14250`.
 * `apm-server.jaeger.http.host` defaults to `localhost:14268`.
 
+[[jaeger-configure-sampling]]
+*Configure Sampling*
+
+The gRPC endpoint supports probabilistic sampling which can be used to reduce the amount of data that your agents collect and send.
+Probabilistic sampling makes a random sampling decision based on the value of `sampling.default`.
+For example, a value of `.2` means that 20% of traces will be sampled.
+Sampling ratios can also be adjusted, on a per service basis, via the Agent configuration page in the APM app.
+
+To enable sampling:
+
+* Set `apm-server.grpc.sampling.enabled` to `enabled`.
+* Set `apm-server.grpc.sampling.default` to the desired sampling rate (between `0` and `1.0`).
```

Comment:
I removed the config parameters, based on https://github.com/elastic/apm-server/pull/3490#discussion_r393410810. The gRPC sampling endpoint will be enabled as soon as `apm-server.jaeger.grpc.enabled: true`.